CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.55.1
```

Comment:
1.55.0 hasn't been released yet

Reply:
Should I release this as 1.55.0 then, or should I make it a -dev version while we wait for more changes?